docker-compose-tests.yml

```diff
@@ -0,0 +1,31 @@
+version: '2.1'
+
+services:
+
+  test:
+    extends:
+      file: docker-compose.yml
+      service: base-app
+    command: yarn test-watch
+    volumes:
+      - c2s-test-node_modules:/code/node_modules
+      - .:/code
+
+  test-ci:
+    extends:
+      file: docker-compose.yml
+      service: base-app
+    command: yarn run test-ci
+    user: root
+    volumes:
+      - c2s-test-ci-node_modules:/code/node_modules
+    environment:
+      COVERALLS_REPO_TOKEN: ${COVERALLS_REPO_TOKEN}
+      TRAVIS: ${TRAVIS}
+      TRAVIS_BRANCH: ${TRAVIS_BRANCH}
+      TRAVIS_COMMIT: ${TRAVIS_COMMIT}
+      TRAVIS_JOB_ID: ${TRAVIS_JOB_ID}
+
+volumes:
+  c2s-test-node_modules:
+  c2s-test-ci-node_modules:
```

Comment:
The volume name could be something more specific to the repo or possibly more generic e.g. `test-ci-node_modules`?